e2e/playwright/point-click.spec.ts

```diff
@@ -3518,9 +3548,46 @@ radius = 8.69
       const lineCodeToSelection = `|> angledLine([0, 202.6], %, $rectangleSegmentA001)`
       await page.getByText(lineCodeToSelection).click()
       await cmdBar.progressCmdBar()
+      await cmdBar.progressCmdBar()
+      await cmdBar.progressCmdBar()
 
-      const newCodeToFind = `revolve001 = revolve(sketch002, angle = 360, axis = getOppositeEdge(rectangleSegmentA001)) `
-      expect(editor.expectEditor.toContain(newCodeToFind)).toBeTruthy()
+      // const newCodeToFind = `revolve001 = revolve(sketch002, angle = 360, axis = getOppositeEdge(rectangleSegmentA001)) `
+      const newCodeToFind = `revolve001 = revolve(sketch002, angle = 360, axis = rectangleSegmentA001)`
```

Comment:
btw we can do snapshot matching on code in the editor. `toMatchSnapshot` can take any text https://github.com/KittyCAD/modeling-app/pull/5814/files#diff-ddb24425342bbb0c0e1ccec59eab89687f1019d563955d836d61a2ca9f7bdf7bR175